Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.6
+
+WORKDIR /response_operations_ui
+COPY . /response_operations_ui
+EXPOSE 8085
+RUN pip3 install pipenv==8.3.1 && pipenv install --deploy --system
```

Comment:
Could start using the latest version (9.0.1) here.

Reply:
Is there much value in pinning the version if we always want to use the latest? As it's for local development there is less risk in this. I don't see this being updated often otherwise

Reply:
I guess the only issue that could arise is if the flags (--deploy; --system) are changed in future versions but like you said this would be picked up in development.